sdk/appconfiguration/app-configuration/test/public/index.readonlytests.spec.ts

```diff
@@ -71,10 +71,10 @@ describe("AppConfigurationClient (set|clear)ReadOnly", () => {
   });
 
   // Skipping all "accepts operation options flaky tests" https://github.com/Azure/azure-sdk-for-js/issues/26447
-  it.skip("accepts  operation options", async function () {
+  it("accepts  operation options", async function () {
     // Recorder checks for the recording and complains before core-rest-pipeline could throw the AbortError (Recorder v2 should help here)
```

Comment:
still applicable?

Reply:
When I run the tests locally, they pass so I want to unskip them to run a live pipeline test. I'll wait for the public endpoint to confirm this